harvard-university-for-the-creative-arts.csl

```diff
@@ -6,47 +6,51 @@
     <id>http://www.zotero.org/styles/harvard-university-for-the-creative-arts</id>
     <link href="http://www.zotero.org/styles/harvard-university-for-the-creative-arts" rel="self"/>
     <link href="http://www.zotero.org/styles/harvard-anglia-ruskin-university" rel="template"/>
-    <link href="http://community.ucreative.ac.uk/Harvard-Referencing" rel="documentation"/>
+    <link href="http://www.uca.ac.uk/library/academic-support/harvard-referencing/" rel="documentation"/>
     <author>
       <name>Ian Badger</name>
       <email>ibadger@ucreative.ac.uk</email>
     </author>
     <category citation-format="author-date"/>
     <category field="generic-base"/>
     <summary>University for the Creative Arts Harvard style</summary>
-    <updated>2017-02-07T11:38:09+00:00</updated>
+    <updated>2017-06-08T09:18:29+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
+  <locale>
+    <terms>
+      <term name="accessed">accessed on </term>
```

Comment:
@ibadger, we prefer that all terms are defined without any separating punctuation, like the trailing space here. It's better to move that over to where the term is used.